### DISCUSSION TITLE
Welcome to Tines discussions!

### DISCUSSION THREAD

**Discussion by larrykollar** [larrykollar/tines#22] [General]
## 👋 Welcome!

Discussions is a place to connect with other Tines users. We hope that you:

  * Ask questions you’re wondering about. What new features do you need? 
  * Share ideas, especially when it comes to cool additions to `.tinesrc`. Post blog links about how you use Tines to get stuff done, or share your clever extensions.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember, this is a community. We build together 💪.

To get started, add a comment. Introduce yourself, and tell us about what you do with this community.

**Comment by iggu** [comment]
Hello, Larry!

I was investigating the area of terminal outliners and discovered that's almost nothing changed since hnb with them.
After using it after some time I have found your wonderful project and would love to switch to using tines instead of hnb.
But a few issues stop me.

First of all, hnb out of the box, from apt-debs, supports (in a horrible way - but at least) utf8 non-english letters. Tines not.
Changelog says that you at least tried to improve unicode support, but I see that it's worse when in hnb.
What could I do wrong on configure/build step?

Second, what's wrong with undo? Did you just rebound C-z to 'stop' command or got rid of undo totally?

I have alot of ideas how to improve the ideas of hnb with a few effort, but support for unicode (at least partial) is critical and I dont want to continue if it is broken. Undo (even in that ugly way hnb provided it) is too very important.

Thanks in advance. Hope to hear from you soon.

**Reply by larrykollar** [reply]
Hi, sorry for being so late with this.

My evil master plan for getting UTF-8 to work is to embed the latest ncurses into the source code. I believe this will also solve problems that Arch Linux users see with Tines.
